src/base/projects/Header.svelte

```diff
@@ -323,12 +334,8 @@
       {/if}
     </div>
   </span>
-  <div class="stat commit-count" on:click={switchContent}>
-    {#if content == ProjectContent.Browser}
-      <strong>{tree.stats.commits}</strong> commit(s)
-    {:else}
-      <strong>Back to Browser</strong>
```

Comment:
@cloudhead I think it's not that clear that one can navigate back by clicking again the commits count.
Should we rely on the browser back button or give the user some other way to do it?

Reply:
Yeah I think eventually there should be several ways, but for now this is kind of a hidden feature anyway. What I'd like to see is:

1) back button
2) click on project name (will work for free once URL changes when commits is clicked)
3) click on the commits button again (current behavior)

Reply:
Ok great, yeah I think that makes sense, will provide it with separate PR